src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -57,6 +57,20 @@ internal static class ProtocolConversions
             { WellKnownTags.NuGet, LSP.CompletionItemKind.Text }
         };
 
+        public static LSP.TextDocumentPositionParams PositionToTextDocumentPositionParams(int position, SourceText text, Document document)
```

Comment:
methods in this file. should we all convert to extension method?

Reply:
in a previous PR @tmat requested they not be extensions.
https://github.com/dotnet/roslyn/pull/34794#discussion_r274064637